### PR TITLE
feat(Backloggd): add presence

### PR DESCRIPTION
**Pull request by Lavenderleaff**
## Description 

This is a presence for Backloggd that resolves [this](https://github.com/PreMiD/Presences/issues/6465) issue.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image

</details>

### CODE REVIEW

File: websites/B/Backloggd/presence.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+const presence = new Presence({
+		clientId: "1003092862285651968",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		startTimestamp: browsingTimestamp,
+		largeImageKey: "https://i.imgur.com/SUZJZEX.png",
+	};
```
Comment (DarkVillager) [inline websites/B/Backloggd/presence.ts:10]:
```suggestion
	},
	{ pathname } = document.location;
```

File: websites/B/Backloggd/presence.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	if (document.location.pathname === "/")
```
Comment (DarkVillager) [inline websites/B/Backloggd/presence.ts:11]: Do this for all 
```suggestion
	if (pathname === "/")
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/B/Backloggd/presence.ts (hunk 2 above)
Comment (Lavenderleaff) [reply websites/B/Backloggd/presence.ts:11]: Idk man

File: websites/B/Backloggd/presence.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+				.querySelectorAll("#game-profile #game-body .sub-title a")[1]
+				.textContent.trim()}`;
+			presenceData.largeImageKey = `${
+				document.querySelector<HTMLImageElement>(
+					".game-cover .overflow-wrapper >img"
+				).src
+			}`;
+		}
+	} else if (pathname.includes("/search/")) {
+		presenceData.details = "Searching 🔎";
```
Comment (DarkVillager) [inline websites/B/Backloggd/presence.ts:30]: Upload the content of [this](https://github.com/PreMiD/Presences/tree/main/.resources) to [the portal](https://discord.com/developers/applications) at rich presence of the current application
```suggestion
		presenceData.details = "Searching";
		presenceData.smallImageKey = "search";
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/B/Backloggd/presence.ts (hunk 3 above)
Comment (Lavenderleaff) [reply websites/B/Backloggd/presence.ts:30]: Thanks but no thanks, Encrypted prefers Imgur Links. Pretty sure emojis are allowed

Comment (DarkVillager) [reply websites/B/Backloggd/presence.ts:30]: I see imgur links is also fine but you need that specific icon from the folder with images.

File: websites/B/Backloggd/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.7",
+    "author": {
+      "name": "Lavenderleaff",
+      "id": "988801425196867644"
+    },
+    "service": "Backloggd",
+    "description": {
+      "en": "Discover, Collect, Analyze your Games",
+      "fr": "Découvrez, Collectez, Analysez vos Jeux"
```
Comment (EncryptedDev) [inline websites/B/Backloggd/dist/metadata.json:10]: @Kozou4ever

**Review by Kozou4ever — APPROVED** [review]

File: websites/B/Backloggd/presence.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+		presenceData.state = `${document
+			.querySelector("#search-title h1")
+			.textContent.trim()}`;
```
Comment (EncryptedDev) [inline websites/B/Backloggd/presence.ts:33]: Use smallImageKey for the emoji instead

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/B/Backloggd/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.7",
+    "author": {
+      "name": "Lavenderleaff",
+      "id": "988801425196867644"
+    },
+    "service": "Backloggd",
+    "description": {
+      "en": "Discover, Collect, Analyze your Games",
```
Comment (DarkVillager) [inline websites/B/Backloggd/dist/metadata.json:9]: image
```suggestion
      "en": "Keep a virtual backlog of your video game collection, then rate and review the ones you've played to share with your friends!",
```

File: websites/B/Backloggd/presence.ts [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+	else if (pathname.startsWith("/about/")) {
+		if (pathname.endsWith("terms-of-service"))
+			presenceData.details = "Viewing the Terms of Service";
+		else if (pathname.endsWith("privacy"))
+			presenceData.details = "Viewing the Privacy Policy";
+	} else if (pathname.startsWith("/users/")) {
+		if (pathname.endsWith("sign_in")) presenceData.details = "Signing In";
+		else if (pathname.endsWith("sign_up")) presenceData.details = "Signing Up";
+		else if (pathname.endsWith("password/new"))
+			presenceData.details = "Resetting Password";
```
Comment (DarkVillager) [inline websites/B/Backloggd/presence.ts:55]: Privacy concerns? @EncryptedDev

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/B/Backloggd/presence.ts (hunk 7 above)
Comment (Lavenderleaff) [reply websites/B/Backloggd/presence.ts:56]: > Privacy concerns? @EncryptedDev

Removed it, calm down, geez

**Review by EncryptedDev — APPROVED** [review]
Other than this, LGTM

File: websites/B/Backloggd/presence.ts [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+		else if (pathname.endsWith("sign_up")) presenceData.details = "Signing Up";
+		else presenceData.details = "Viewing an unsupported page.";
+	} else if (pathname.startsWith("/u/")) {
+		presenceData.details = `Viewing ${document
+			.querySelector("#profile-header h3")
+			.textContent.trim()}'s Profile`;
+		presenceData.largeImageKey = `${
+			document.querySelector<HTMLImageElement>("#profile-header img").src
+		}`;
+	} else presenceData.details = "Viewing an unsupported page.";
```
Comment (EncryptedDev) [inline websites/B/Backloggd/presence.ts:62]:
```suggestion
	} else presenceData.details = "Viewing an unsupported page";
```

File: websites/B/Backloggd/presence.ts [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+	else if (pathname.startsWith("/about/")) {
+		if (pathname.endsWith("privacy/"))
+			presenceData.details = "Viewing the Privacy Policy";
+		else if (pathname.endsWith("terms-of-service/"))
+			presenceData.details = "Viewing the Terms of Service";
+		else presenceData.details = "Viewing the About Page";
+	} else if (pathname.startsWith("/users/")) {
+		if (pathname.endsWith("sign_in")) presenceData.details = "Signing In";
+		else if (pathname.endsWith("sign_up")) presenceData.details = "Signing Up";
+		else presenceData.details = "Viewing an unsupported page.";
```
Comment (Bas950) [inline websites/B/Backloggd/presence.ts:54]:
```suggestion
		else presenceData.details = "Viewing an unsupported page";
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**